docs/guides/admin/tracking.md

```diff
@@ -1,31 +1,33 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-## introduction
+# Understanding Tracking
+
+## Introduction
 SeAT implements the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/). This document aims to explain in as much detail as possible how it has been implemented in SeAT, as well as what is tracked and what is not. This document aims to be as transparent as possible.
 
-## whyyyyyyy
+## Why
 Well, the most obvious is it being nice to know how much SeAT is actually being used. One may argue that Github & packagist gives statistics on how many times the project has been installed / cloned, but that does not really reflect how many *actual* active installations there are.
 
 Knowing how many active installs there are, encourages development.
 
 Lastly, certain exception types are also sent as hits. This helps *immensely* with figuring out if a new version may have a serious bug.
 
-## how its implemented
+## How its implemented
 First of all, the Google Analytics Measurement Protocol is really just that. It just measures usage. In summary, when certain events happen, only a hit with what happened is sent. **No other data is sent with the hit.**
 
 For example. When the scheduler queues jobs, a hit is sent that says that this happened, and that it happened for *x* amount of keys. This can be seen in the following line of code: [https://github.com/eveseat/console/blob/master/src/Commands/Eve/QueueKeys.php#L96-L103](https://github.com/eveseat/console/blob/master/src/Commands/Eve/QueueKeys.php#L96-L103). It can also be seen that no other data goes along with the hit. For example, the `key_id` and `vcode` (which is what most will be worried about) does not go with the hit.
```

Comment:
Refrence to `key_id` and `vcode` should be removed.
Maybe add that no refreshtokens are being shared and that this would not help anyone without knowing the ESI-Secrent between Application and ESI Portal